hrui/js/hrui.js

```diff
@@ -12,7 +12,23 @@
       }
     });
   }
+
+  /**
+   * Update label 'for' attr to works with the datepicker
+   *
+   * @param  {jQuery Obejct} line [tr parent element]
+   */
+  function updateLabelFor(line) {
```

Comment:
This function name is extremely generic given the very specific job that it does (updating the `[for]` attr to have the same value as the `[id]` attr of the datepicker field)

Maybe something like:

```js

/**
 * ....
 *
 * @param {jQuery object} input the input field the label should linked to
 */
function linkLabelToDatepickerInput($input) {

}

```
and also, why passing the entire line to the function? If you know already what element you're looking for, why not passing it directly to the function (like above)?